src/OrchardCore.Modules/OrchardCore.Workflows/Http/Controllers/HttpWorkflowController.cs

```diff
@@ -143,7 +157,7 @@ public async Task<IActionResult> Invoke(string token)
                 {
                     if (_logger.IsEnabled(LogLevel.Warning))
                     {
-                        _logger.LogWarning("No workflow found that is blocked on activity '{ActivityId}'", startActivity.ActivityId);
+                        _logger.LogWarning(S["No workflow found that is blocked on activity '{ActivityId}'"], startActivity.ActivityId);
```

Comment:
I think we should use params instead, but wait for @sebastienros to confirm

Reply:
The code before localization was perfect.